qmcpy/discrete_distribution/lattice/lattice.py

```diff
@@ -40,6 +50,24 @@ class Lattice(LD):
            [0.5 , 0.5 ],
            [0.25, 0.75],
            [0.75, 0.25]])
+    >>> Lattice(dimension=2,randomize=False,seed=182,generating_vector = 23).gen_samples(8, warn=False)
+    array([[0.   , 0.   ],
+           [0.5  , 0.5  ],
+           [0.25 , 0.25 ],
+           [0.75 , 0.75 ],
+           [0.125, 0.125],
+           [0.625, 0.625],
+           [0.375, 0.375],
+           [0.875, 0.875]])
+    >>> Lattice(dimension=4,randomize=False,seed=353,generating_vector = 26).gen_samples(8,warn=False)
```

Comment:
remove whitespace when possible e.g. `generating_vector = 26` should be `generating_vector=26`. This is nit-picky, but we want our code to look as uniform as possible